pkg/distributor/otel.go

```diff
@@ -209,7 +215,7 @@ func OTLPHandler(
 }
 
 func otlpHandler(
-	maxRecvMsgSize int,
+	maxOtelDecompressedRecvMsgSize int,
```

Comment:
Same comment as in `OTLPHandler` applies here.

Reply:
renaming is done in commit https://github.com/grafana/mimir/commit/1ba1bcb4854d016dfada97077fc1696ea23c84e3